docs/manage/data/label.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Label Data"
+linkTitle: "Label Data"
+description: "Label image data on the DATA page in the Viam Cloud."
+weight: 39
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+images: [ "/manage/data/img/label-dog.gif" ]
+# SME: Tahiya Salam and Alexis Wei
+---
+
+To label data in a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+{{< alert title="Important" color="note" >}}
```

Comment:
Should "Important" be changed to one of our standard style guide titles such as "Note"? Or added to the style guide?

Reply:
I think actually added to the style guide or maybe replace the Note in the style guide event. I'm never entirely sure what the implication for a note should be - does that tell the user "definitely read this" - or "read this for extra information". Let's discuss in team slack.

Reply:
nvm let's just change it to note

Reply:
Haha okay either way!